scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/StaxJobFactory.java

```diff
@@ -208,10 +197,33 @@ private Job createJob(File file, Map<String, String> replacementVariables,
         }
     }
 
+    private Job createJobFromInputStream(InputStream inputStream, Map<String, String> replacementVariables,
+            Map<String, String> replacementGenericInfos) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)